### ISSUE TITLE
Make dnf rules apply to microdnf

### ISSUE THREAD

**Issue by carlsmedstad** [hadolint/hadolint#742]
- [ ] This is a bug report
- [x] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

`microdnf` is a slimmed-down version of `dnf` used in RedHat's minimal UBI images. Make rules for `dnf` apply to `microdnf`.

This Dockerfile:

```Dockerfile
FROM registry.access.redhat.com/ubi7/ubi-minimal:7.9

RUN microdnf install \
    rh-python38
```

should produce the same errors as this one:

```Dockerfile
FROM registry.access.redhat.com/ubi7/ubi-minimal:7.9

RUN dnf install \
    rh-python38
```

### Actual behavior

No errors are produced for non-ideal usage of `microdnf`.
